src/Game.js

```diff
@@ -0,0 +1,20 @@
+class Game {
+  constructor(){
+    this.frameScores = []
+  }
+
+  addFrame(frame) {
+    if (frame.isOpen() === false) { game.push(frame.getScore())
+    }
+  }
+
+  // need to implement this - perhaps it moves to frame class
+
+  getScore() {
```

Comment:
I'm not sure I follow what 'a' and 'b' are here, are they the first and second rolls of a frame?